src/Templates/src/templates/maui-blazor/MauiApp.1.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
 
     <PropertyGroup>
-        <TargetFrameworks>DOTNET_TFM-android;DOTNET_TFM-ios;DOTNET_TFM-maccatalyst</TargetFrameworks>
+        <TargetFrameworks>DOTNET_TFM-android</TargetFrameworks>
+        <TargetFrameworks Condition=" ! $([MSBuild]::IsOSPlatform('linux'))">$(TargetFrameworks);DOTNET_TFM-ios;DOTNET_TFM-maccatalyst</TargetFrameworks>
         <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);DOTNET_TFM-windows10.0.19041.0</TargetFrameworks>
```

Comment:
I feel that this block of conditions is getting incredibly hard to understand. There's an always-on Android. There's a negative condition ios/maccat, and a positive-condition Windows.

Is there a cleaner way to slice these up?

Reply:
Yes, see https://github.com/dotnet/maui/pull/16143

We need to fix the SDK.

Reply:
Yeah I'm wary of making these changes to the template to work around the correct fix to the problem.

Reply:
> Is there a cleaner way to slice these up?

Depends on definition of "cleaner". 

If it means "easier to understand" then maybe:

```
        <TargetFrameworks Condition=" $([MSBuild]::IsOSPlatform('windows')) || $([MSBuild]::IsOSPlatform('osx'))">$(TargetFrameworks);DOTNET_TFM-ios;DOTNET_TFM-maccatalyst</TargetFrameworks>
```

I went for less-cycles concept driven by "green", "sustainable", "shorter inner loop" (for large numbers especially).

Note "windows" as 1st in most cases 2nd condition should not be evaluated (only for mac).

Reply:
> Yes, see #16143

Matthew. I really did not see that PR, but it seems similar, moreover the same if whitespace is ignored.

https://github.com/dotnet/maui/blob/37da7d68c414f352e2b258d3d565aaec9227d88f/src/Templates/src/templates/maui-blazor/MauiApp.1.csproj#L5

https://github.com/dotnet/maui/blob/37da7d68c414f352e2b258d3d565aaec9227d88f/src/Templates/src/templates/maui-lib/MauiLib1.csproj#L5